### PR TITLE
wip chore(build): travis container build

### PR DESCRIPTION
**Pull request by vladikoff**
Just testing, WIP

### CODE REVIEW

**Comment by vladikoff** [comment]
We need to figure out how to install libgmp without `sudo` on Travis.
Currently getting this error `../bigint.cc:9:17: fatal error: gmp.h: No such file or directory`

File: .travis.yml [hunk 1]
```diff
@@ -7,28 +7,28 @@ env:
...
 install:
   # install everything for full dev in the fxa-content-server.
-  - travis_retry npm install --silent
+  - travis_retry npm install
   # copy over the configuration that can be used to start the server.
   - cp server/config/local.json-dist server/config/local.json
   # install the resources necessary for the auth server.
   - git clone git://github.com/mozilla/fxa-auth-server.git --depth 1
   - cd fxa-auth-server
-  - freight
```
Comment (zaach) [inline .travis.yml]: :O

File: .travis.yml [hunk 2]
```diff
@@ -7,28 +7,28 @@ env:
...
 before_install:
   - npm install -g npm@2.1
-  - sudo apt-get install libgmp3-dev
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - wget http://selenium-release.storage.googleapis.com/2.43/selenium-server-standalone-2.43.1.jar
   - java -jar selenium-server-standalone-2.43.1.jar &> /dev/null &
-  - export FREIGHT_URL=https://freight.dev.lcip.org
-  # See https://github.com/mozilla/fxa-content-server/issues/1483
-  - npm config set spin false
```
Comment (pdehaan) [inline .travis.yml]: Did you want to nuke the npm spinner bit?

File: .travis.yml [hunk 3]
```diff
@@ -7,28 +7,28 @@ env:
...
-  - npm install -g vladikoff/freight#v0.3.0 && freight
 
 install:
   # install everything for full dev in the fxa-content-server.
-  - travis_retry npm install --silent
+  - travis_retry npm install
```
Comment (pdehaan) [inline .travis.yml]: Why remove the `--silent` flag here?

Comment (vladikoff) [inline .travis.yml]: > Just testing, WIP 

:dancers:

**Pull request closed without merging**